od-czego-zaczac/README.md

```diff
@@ -6,21 +6,21 @@ Przed zdecydowaniem się na rozpoczęcie swojej przygody z testowaniem, warto za
 
 Warto pamiętać o tym, że od paru już lat panuje coraz większe zainteresowanie pracą na rynku IT. Przełożyło się to na ogromną popularność wszelkiego rodzaju szkoleń, kursów czy bootcampów.
 
-Niejednokrotnie można znaleźć artykuły traktujące o tym, że w świecie IT jest ogromne zapotrzebowanie na przykład na programistów. Na grupach facebookowych z ogłoszeniami o pracę można też zaobserwować, że wiele firm szuka specjalistów od testowania niejednokrotnie 2 lata i dłużej, nie potrafiąc zapełnić wakatu.
+Można znaleźć bardzo wiele artykułów traktujących o tym, że w świecie IT jest ogromne zapotrzebowanie na przykład na programistów. Również na grupach facebookowych z ogłoszeniami o pracę można zaobserwować, że wiele firm szuka specjalistów od testowania niejednokrotnie 2 lata i dłużej, nie potrafiąc zapełnić wakatu.
```

Comment:
"bardzo wiele" jest jednak drobnym wyolbrzymieniem :)

Reply:
Może coś bardziej neutralnego. Coś jak:
"W internecie można trafić na artykuły traktujące o tym, że w świecie IT jest ogromne zapotrzebowanie na przykład na programistów."

Reply:
Changed